### PR TITLE
KIK-8314 wipe all accounts from device

### PR DESCRIPTION
**Pull request by shayb**
recursively delete all keystore directory and its content
adding test

### CODE REVIEW

File: kin-sdk-core/src/main/java/kin/sdk/core/KinClient.java [hunk 1]
```diff
@@ -100,6 +100,10 @@ public void deleteAccount(String passphrase) throws DeleteAccountException {
         }
     }
 
+    public void wipeoutAccount() {
```
Comment (berryve) [inline kin-sdk-core/src/main/java/kin/sdk/core/KinClient.java]: add javadoc description

File: kin-sdk-core/src/main/java/kin/sdk/core/EthClientWrapper.java [hunk 2]
```diff
@@ -43,11 +44,12 @@
     EthClientWrapper(android.content.Context androidContext, ServiceProvider serviceProvider)
         throws EthereumClientException {
         this.serviceProvider = serviceProvider;
+        this.androidContext = androidContext.getApplicationContext();
```
Comment (YohayBar) [inline kin-sdk-core/src/main/java/kin/sdk/core/EthClientWrapper.java]: I prefer not to save the context - there is no need to right?  wipeoutAccount can get context as well

File: kin-sdk-core/src/main/java/kin/sdk/core/EthClientWrapper.java [hunk 3]
```diff
@@ -108,15 +100,35 @@ private void initKeyStore(android.content.Context context) throws EthereumClient
...
+        return new StringBuilder(androidContext.getFilesDir().getAbsolutePath())
+            .append(File.separator)
+            .append("kin")
+            .append(File.separator)
+            .append("keystore");
+    }
+
+    public String getKeyStoreFullPath() {
+        return getKeyStoreBasePath().append(File.separator)
+            .append(serviceProvider.getNetworkId())
```
Comment (YohayBar) [inline kin-sdk-core/src/main/java/kin/sdk/core/EthClientWrapper.java]: why do we need. both getKeyStoreFullPath and getKeyStoreBasePath? we can just whip the account in current client network id

File: kin-sdk-core/src/main/java/kin/sdk/core/EthClientWrapper.java [hunk 4]
```diff
@@ -122,11 +122,12 @@ public void deleteAccount(Account account, String passphrase) throws DeleteAccou
         }
     }
 
-    public void wipeoutAccount() {
+    public void wipeoutAccount() throws EthereumClientException {
         File keystoreDir = new File(getKeyStoreBasePath().toString());
         if (keystoreDir.exists()) {
             deleteRecursive(keystoreDir);
         }
+        initKeyStore();
```
Comment (YohayBar) [inline kin-sdk-core/src/main/java/kin/sdk/core/EthClientWrapper.java]: why is this needed?

File: kin-sdk-core/src/main/java/kin/sdk/core/EthClientWrapper.java (hunk 2 above)
Comment (berryve) [reply kin-sdk-core/src/main/java/kin/sdk/core/EthClientWrapper.java]: we're just saving application context - that is ok

File: kin-sdk-core/src/main/java/kin/sdk/core/EthClientWrapper.java (hunk 4 above)
Comment (berryve) [reply kin-sdk-core/src/main/java/kin/sdk/core/EthClientWrapper.java]: I just added it otherwise geth returns the last account even though we've wiped out the directory
